### ISSUE TITLE
Reduce threshold on the answering functions

### ISSUE THREAD

**Issue by neelpawarcmu** [touretzkyds/BERT-insight#5]
The current threshold is stringent and gives 'no predictions' for quite a lot of cases. See screenshot below. Note that the logits in the screenshot do indicate the correct answer, but there is some thresholding that disqualifies them from being answers to the question. 

There should be a function that performs these operations under the hood in the source code. Find out the same and reduce the threshold. In future versions, we can add a slider that allows the user to pick a threshold.

![image](https://user-images.githubusercontent.com/81655816/230806301-ef9a4caf-d496-4840-abdb-a7175a9d797d.png)
